react-common/components/controls/EmbedVideo.tsx

```diff
@@ -1,45 +1,67 @@
-import * as React from "react";
+
 import { classList, ControlProps } from "../util";
+import { useRef, useEffect } from 'react';
+import * as React from 'react';
+import { MediaPlayer } from "dashjs";
 
 export interface EmbedVideoProps extends ControlProps {
-  src: "youtube";
-  id: string;
-  autoplay?: boolean;
-  allowFullScreen?: boolean;
+    src: "youtube" | "streams";
```

Comment:
Lets not call it streams. No one knows this other than us. Call it azuremedia

Reply:
makes sense, thanks for feedback!